Resources/Prototypes/Entities/Objects/Misc/kudzu.yml

```diff
@@ -208,3 +208,58 @@
         types:
           Asphyxiation: -0.25
 
+- type: entity
```

Comment:
why does this not extend regular kudzu

Reply:
because kudzu has a lot of components that I don't need. Damage resistance parameters, barotrauma and others

Reply:
might be worth putting the things they have in common in an abstract prototype for them to share